ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -158,12 +181,37 @@ public struct MSFButtonView: View {
         })))
         buttonView.tokens.windowProvider = self
         view.backgroundColor = UIColor.clear
+        setupLargeContentViewer()
     }
 
-    var window: UIWindow? {
-        return self.view.window
+    private func setupLargeContentViewer() {
+        let largeContentViewerInteraction = UILargeContentViewerInteraction()
+        view.addInteraction(largeContentViewerInteraction)
+        largeContentViewerInteraction.gestureRecognizerForExclusionRelationship.delegate = self
+        view.scalesLargeContentImage = true
+        view.showsLargeContentViewer = shouldEnableLargeContentViewerForOSVersion || buttonView.tokens.style.isFloatingStyle
+
+        imagePropertySubscriber = buttonView.state.$image.sink { buttonImage in
+            self.view.largeContentImage = buttonImage
+        }
+
+        textPropertySubscriber = buttonView.state.$text.sink { buttonText in
+            self.view.largeContentTitle = buttonText
+        }
     }
 
+    private let shouldEnableLargeContentViewerForOSVersion: Bool = {
+        if #available(iOS 14.0, *) {
+            return false
+        } else {
+            return true
```

Comment:
although this is just temporary right? can we have a button with increased text based on text setting on iOS 13?

Reply:
Yes. As per our conversation today, I'll work on finding a solution to make that happen in iOS 13 as well (except for the floating style which should have a fixed non-scalable text).